CMakeLists.txt

```diff
@@ -67,8 +75,8 @@ if (CMAKE_BUILD_TYPE MATCHES "Debug|debug|DEBUG")
   list(APPEND _glvis_compile_defs "GLVIS_DEBUG")
 endif()
 
-# Base everything on MFEM
-find_package(MFEM REQUIRED)
+# Include paths and libraries needed by MFEM
+list(APPEND CMAKE_CXX_FLAGS "${MFEM_CXX_FLAGS}")
```

Comment:
I don't think `CMAKE_CXX_FLAGS` is actually a list. If you prefer to set the flags globally, it should be

    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${MFEM_CXX_FLAGS}")

The alternative is to add them to the list called `_glvis_compile_opts`, which will be added to each target.